.ci/install.sh

```diff
@@ -12,34 +28,44 @@ if [ ! -f "${TRAVIS_BUILD_DIR}/install/bin/luarocks" ]; then
         cmake . -DWITH_LUAJIT21=ON -DCMAKE_INSTALL_PREFIX="${TRAVIS_BUILD_DIR}/install"
         make install
     popd
+else
+    echo "Using cached luarocks."
 fi
 
-mkdir "${HOME}/.luarocks"
-cp "${TRAVIS_BUILD_DIR}/install/etc/luarocks/config.lua" "$HOME/.luarocks/config.lua"
-echo "wrap_bin_scripts = false" >> "$HOME/.luarocks/config.lua"
-travis_retry luarocks --local install luafilesystem
-# for verbose_print module
-travis_retry luarocks --local install ansicolors
-travis_retry luarocks --local install busted 2.0.rc12-1
-#- mv -f $HOME/.luarocks/bin/busted_bootstrap $HOME/.luarocks/bin/busted
-travis_retry luarocks --local install luacov
-# luasec doesn't automatically detect 64-bit libs
-travis_retry luarocks --local install luasec OPENSSL_LIBDIR=/usr/lib/x86_64-linux-gnu
-travis_retry luarocks --local install luacov-coveralls --server=http://rocks.moonscript.org/dev
-travis_retry luarocks --local install luacheck
-travis_retry luarocks --local install lanes  # for parallel luacheck
+if [ ! -d "${HOME}/.luarocks" ]; then
```

Comment:
we can move those install commands into a separate shell file and use md5sum of the file as suffix for the local luarocks directory.